test/unit/feeds.test.ts

```diff
@@ -0,0 +1,126 @@
+import { ParsingError, SubmitFeedParameters } from '../../src'
+import { NextToken } from '../../src/parsing'
+import { createMockHttpClient, mockMwsFail, parsingError } from '../utils'
+
+describe('feeds', () => {
+  describe('submitFeed', () => {
+    const parameters: SubmitFeedParameters = {
+      FeedContent: '',
+      FeedType: '_POST_PRODUCT_DATA_',
+    }
+
+    it('returns feed info of submitted feeds if succesful', async () => {
+      expect.assertions(1)
+
+      const mockSubmitFeed = createMockHttpClient('feeds_submit_feed')
+
+      expect(await mockSubmitFeed.feeds.submitFeed(parameters)).toMatchSnapshot()
+    })
+
+    it('throws a parsing error when the response isnt valid', async () => {
+      expect.assertions(1)
+
+      await expect(() => mockMwsFail.feeds.submitFeed(parameters)).rejects.toStrictEqual(
+        new ParsingError(parsingError),
+      )
+    })
+  })
+
+  describe('getFeedSubmissionResult', () => {
+    const parameters = { FeedSubmissionId: '' }
+
+    it('returns an XML file when succesful', async () => {
```

Comment:
Docs say that's what it returns

http://docs.developer.amazonservices.com/en_CA/feeds/Feeds_GetFeedSubmissionResult.html

Not really sure how to handle returning XML files so I just made it return the XML as a string

Reply:
My thought process was since it's an XML file that's specific to the 'feed' a user is requesting, the user might want the XML string more than a JS object